packages/legacy-workbench/src/js/angular/guides/steps/complex/select-repository/plugin.js

```diff
@@ -38,7 +47,8 @@ PluginRegistry.add('guide.step', [
                             throw (error);
                         }),
                     show: (guide) => () => {
-                        $('#repositorySelectDropdown').addClass('autoCloseOff');
+                        setRepositorySelectorAutoClose(false);
+                        $('.onto-repository-selector').addClass('auto-close-off');
```

Comment:
wasn't the goal of the task to be removed usage of angular utility functions and jquery?

Reply:
Removed